### DISCUSSION TITLE
Save passwords in your browser

### DISCUSSION THREAD

**Discussion by bceda** [security-mountain/security-mountain.github.io#4] [Ideas]
This is a controversial one.

Whilst saving passwords in the browser may mean users pick stronger passwords, as there isn't the need to remember it, saving passwords in the browser also poses a risk if the device being used is lost, stolen or compromised.

**Comment by ElliottS200** [comment]
Definitely an interesting one, I think that promoting the usage of a password manager or storing passwords in browser is definitely worth the risk of device loss, especially due to the risks posed by weak passwords. Passwords can always be reset.

**Comment by apollodelphi702** [comment]
I do recall that in the past, despite being a single point of failure (if the store gets compromised all of the user's passwords are divulged) this has been an interesting defense against keyloggers. Perhaps further research is needed?
